DunglasActionBundle.php

```diff
@@ -23,6 +23,11 @@
      */
     public function build(ContainerBuilder $container)
     {
-        $container->addCompilerPass(new RegisterCompilerPass());
+        $passConfig = $container->getCompilerPassConfig();
```

Comment:
What is the rationale behind this change? Maybe it deserves a comment in the code.